### PR TITLE
Simplified and faster package.json

### LINKED ISSUE #43: The current test only link/unlink the -schibsted repo
```
    "pretest": "npm link packages/eslint-config-schibsted",
    "posttest": "npm unlink packages/eslint-config-schibsted",
```

Seems like the process for -react repo is missing?

This whole linking and unlinking may be unnecessary now that #36 is merged.

### LINKED ISSUE #44: Use Lerna
I did a little feasibility test to see if we can use [Lerna](https://lernajs.io/) for handling our two packages.

Turns out [Lerna doesn't support Yarn](https://github.com/lerna/lerna/issues/371) yet. Interestingly Jest from Facebook (the company who made Yarn on the first place [is still using Lerna](https://github.com/facebook/jest/blob/master/lerna.json) *v2.0.0-beta.31* despite using [yarn](https://github.com/facebook/jest/blob/master/yarn.lock)).

Anyway, [there are a bunch of yarn issues](https://github.com/lerna/lerna/issues?utf8=%E2%9C%93&q=yarn) on Lerna that one has to investigate.

My two cents:

* Use the `package.json` in the root for now until we have enough packages that justifies bringing another dependency into the project
* Wait and see if Lerna supports Yarn fully or [Yarn will swallow Lerna](https://github.com/yarnpkg/yarn/issues/946#issuecomment-264597575). Either way, it seems to early to decide.

### PR DESCRIPTION
**Pull request by alexewerlof**
simplify package.json in the root by removing the link/unlink process. Fix #43 
use yarn instead of `npm install` in the 2 subpackages. closes #44 
Added a bit of documentation about how to test the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,14 +6,12 @@
...
-    "preinstall": "npm run install:config && npm run install:config-react",
-    "install:config": "cd packages/eslint-config-schibsted && npm prune && npm install",
-    "install:config-react": "cd packages/eslint-config-schibsted-react && npm prune && npm install",
-    "pretest": "npm link packages/eslint-config-schibsted",
-    "test": "npm run test:schibsted && npm run test:schibsted-react",
-    "posttest": "npm unlink packages/eslint-config-schibsted",
-    "test:schibsted": "cd packages/eslint-config-schibsted && npm test",
-    "test:schibsted-react": "cd packages/eslint-config-schibsted-react && npm test"
+    "install": "npm run install:eslint-config-schibsted && npm run install:eslint-config-schibsted-react",
+    "install:eslint-config-schibsted": "cd packages/eslint-config-schibsted && yarn",
```
Comment (SimenB) [inline package.json]: Don't need the `eslint-config` prefix in the command

**Review by SimenB — CHANGES_REQUESTED** [review]
Not linking will make the react config run against published eslint and not the one in the branch. Is that wanted?

If we go for Lerna, it'll handle that for us (I'll create issue later this week), but might be worhth waiting for lerna/lerna#371

File: README.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ Then add the `extends` option to your `.eslintrc`:
 
 You can override specific settings by specifying them as normal. See <http://eslint.org/docs/developer-guide/shareable-configs> for more details.
 
+## Testing
+
+We use Yarn for installation.
+It is recommended to install Yarn using the native installation method for your environment.
+See https://yarnpkg.com/en/docs/install
+So do a `npm i -g yarn` if you don't have it already.
```
Comment (SimenB) [inline README.md]: Npm install is deprecated for yarn

File: .travis.yml [hunk 3]
```diff
@@ -2,3 +2,5 @@ sudo: false
 language: node_js
 node_js: stable
 cache: yarn
+
+before_install: npm i -g yarn
```
Comment (SimenB) [inline .travis.yml]: Use apt for this

**Review by SimenB — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alexewerlof) [reply package.json]: Thought it'll be more expressive if we mention the full name of the package. But maybe we can stick with the "eslint" package name (that's without `eslint-config-` prefix. Is that what you're aiming for? I'm cool with that.

File: README.md (hunk 2 above)
Comment (alexewerlof) [reply README.md]: Right. Thanks for pointing it out. Will update the instructions 👍

File: .travis.yml (hunk 3 above)
Comment (alexewerlof) [reply .travis.yml]: tx. Will update it 👍

**Comment by alexewerlof** [comment]
> Not linking will make the react config run against published eslint and not the one in the branch. 

@SimenB this is no more true since we directly point to the files in the current repo's .eslint:

````
{
    "extends": [
        "./packages/eslint-config-schibsted/index.js",
        "./packages/eslint-config-schibsted/env-node.js"
    ]
}
````

File: package.json (hunk 1 above)
Comment (SimenB) [reply package.json]: Yes :)

**Comment by SimenB** [comment]
> @SimenB this is no more true since we directly point to the files in the current repo's .eslint

True!

File: package.json (hunk 1 above)
Comment (alexewerlof) [reply package.json]: Changes coming soon 🚚

**Review by SimenB — APPROVED** [review]
LGTM. Please squash on merge :smile:

**Pull request merged**